src/SvgArrow.js

```diff
@@ -233,6 +273,9 @@ const SvgArrow = ({
       <path
         d={pathString}
         style={{ fill: 'none', stroke: strokeColor, strokeWidth, strokeDasharray }}
+        markerStart={
+          enableStartMarker ? `url(${location.href.split('#')[0]}#${arrowMarkerId})` : undefined
```

Comment:
You could've made a variable to refactor with the end marker, right?

```suggestion
          enableStartMarker ? markerUrl : undefined
```

Reply:
True, I'll change it. Everything was mostly quick-and-dirty :)